src/particles/MultiPlasma.cpp

```diff
@@ -43,10 +43,11 @@ MultiPlasma::InitData (int lev, amrex::BoxArray slice_ba,
 amrex::Real
 MultiPlasma::maxDensity ()
 {
-    amrex::Real max_density = 0;
+    amrex::Real max_density = 0.;
     for (auto& plasma : m_all_plasmas) {
         max_density = amrex::max(max_density, plasma.m_density);
     }
+    AMREX_ALWAYS_ASSERT(max_density>0.);
     return amrex::max(max_density, m_adaptive_density);
```

Comment:
```suggestion
    AMREX_ALWAYS_ASSERT(amrex::max(max_density, m_adaptive_density)>0.);
    return amrex::max(max_density, m_adaptive_density);
```

I think it should be asserting the maximum of `plasma.m_density` and `m_adaptive_density`. The `adaptive_density` is used when no plasma is there and `plasma.m_density = 0` (e.g., when we use external fields, as in the adaptive time step CI test)

Reply:
Ah, but not even then this should be asserted here, as this triggers for all simulations without a plasma.
This assert should only be used in the adaptive time step and there we assert for plasma density `!= 0` (this should probably be switched to `> 0`.